docs/bnb_smart_chain/staking/developer-guide.md

```diff
@@ -0,0 +1,247 @@
+
+### Edit Vote Address
+
+To change the vote address of a validator, use the `editVoteAddress` function with the following parameters:
+
+```solidity
+function editVoteAddress(bytes calldata newVoteAddress, bytes calldata blsProof) external
+```
+
+- `newVoteAddress`: The new vote address of the validator.
+- `blsProof`: The BLS signature as proof of the vote address.
+
+## Delegation Operations
+
+### Delegate
+
+To delegate BNB to a validator, call the `delegate` function with the following parameters:
+
+```solidity
+function delegate(address operatorAddress, bool delegateVotePower) external payable
+```
+
+- `operatorAddress`: The operator address of the validator.
+- `delegateVotePower`: The flag to indicate whether the delegator would like to delegate his/her voting power
+  to the validator for governance.
+
+### Undelegate
+
+To undelegate BNB from a validator, use the `undelegate` function with the following parameters:
+
+```solidity
+function undelegate(address operatorAddress, uint256 shares) external
+```
+
+- `operatorAddress`: The operator address of the validator.
+- `shares`: The amount of shares to undelegate from the validator.
+
+### Redelegate
+
+To redelegate BNB from one validator to another, use the `redelegate` function with the following parameters:
+
+```solidity
+function redelegate(address srcValidator, address dstValidator, uint256 shares, bool delegateVotePower) external
+```
+
+- `srcValidator`: The operator address of the source validator to redelegate from.
+- `dstValidator`: The operator address of the destination validator to redelegate to.
+- `delegateVotePower`: The flag to indicate whether the delegator would like to delegate his/her voting power
+  to the destination validator for governance.
+
+## Claim
+
+To claim undelegated BNB after the unbonding period, use the `claim` function for a single request or `claimBatch` for
+multiple requests:
+
+```solidity
+function claim(address operatorAddress, uint256 requestNumber) external
+```
+
+- `operatorAddress`: The operator address of the validator.
+- `requestNumber`: The number of unbonding requests to claim from. `0` means claiming from all unbonding requests.
+
+```solidity
+function claimBatch(address[] calldata operatorAddresses, uint256[] calldata requestNumbers) external
+```
+
+- `operatorAddress`: The operator addresses of the validators.
+- `requestNumber`: The numbers of unbonding requests to claim from the validators.
+
+## FAQs
+
+### What is validator's credit contract?
+
+For each validator, there is a credit contract which will be automatically deployed when it is created.
+Meanwhile, the contract cannot be upgraded or changed by any validator operator.
+
+The credit contract is a BEP20 contract, and the ABI is the same
+as [Stake Credit contract](https://github.com/bnb-chain/bsc-genesis-contract/blob/master/abi/stakecredit.abi).
+
+It provides functions for querying delegations, including:
```

Comment:
<img width="792" alt="image" src="https://github.com/huihzhao/docsite/assets/7310198/9f9b6746-045e-4b7f-8ea5-758909d65c3b">

The format is chaos.

Reply:
fixed